lib/registries/addon/drafts/draft/page/template.hbs

```diff
@@ -72,7 +72,7 @@
                 </layout.leftNav>
                 <layout.main local-class='Main'>
                     <main
-                        {{did-insert draftManager.onPageChange this.pageIndex this.inReview}}
+                        {{did-insert draftManager.onPageChange this.pageIndex this.inReview true}}
```

Comment:
this lonely `true` has no meaning in the template -- would be cleaner to split `onPageChange` into two actions (one for initial load with `did-insert`, one for page change with `did-update`)